```{r, echo=FALSE, eval=TRUE, warning=FALSE, message=FALSE, cache=FALSE}
opts_chunk$set("dev" = c("png", "cairo_pdf"))
opts_chunk$set(results = "hold")
opts_chunk$set(echo = FALSE)
opts_chunk$set(fig.show = "hold")
opts_chunk$set(warning = FALSE)
opts_chunk$set(fig.align = "center")
opts_chunk$set(cache = FALSE)

library("randomForest", quietly=TRUE)
library("png", quietly=TRUE)

set.seed(19760620)
source("code/rf_baseline_analysis.R")
```

**Manipulation of the Gut Microbiota Reveals Role of Gut Microbiota in Colon
Tumorigenesis**

Joseph P. Zackular<sup>1</sup>, Nielson T. Baxter<sup>1</sup>, Grace Y.
Chen<sup>2*</sup>, and Patrick D. Schloss<sup>1*</sup>

\* To whom correspondence should be addressed.

pschloss@umich.edu; gchenry@umich.edu

1 Department of Microbiology and Immunology, University of Michigan, Ann
Arbor, MI

2 Department of Internal Medicine, Division of Hematology and Oncology,
University of Michigan, Ann Arbor, MI




**Abstract**

There is growing evidence that individuals with colonic adenomas and carcinomas
harbor a distinct microbiota. Alterations to the gut microbiota may allow the
outgrowth of bacterial populations that induce genomic mutations or exacerbate
tumor-promoting inflammation. In addition, it is likely that the loss of key
bacterial populations may result in the loss of protective functions that are
normally provided by the microbiota. We explored the role of the gut microbiota
in colon tumorigenesis using an inflammation-based murine model. We observed
that perturbing the microbiota with different combinations of antibiotics did
not change the bacterial load but reduced the number of tumors at the end of the
model. Using the random forest machine learning algorithm we successfully
modeled the number of tumors that developed over the course of the model based
on the composition of the microbiota at the beginning. The timing of antibiotic
treatment was an important determinant of tumor outcome as colon tumorigenesis
was arrested with the use of antibiotics during the inflammation period of the
murine model. Together, these results indicate that it is possible to predict
colon tumorigenesis based on the composition of the microbiota and that altering
the gut microbiota can alter the course of tumorigenesis.


**Keywords:** azoxymethane, dextran sodium sulfate, 16S rRNA gene sequencing,
microbial ecology, microbiome






**Introduction**

The mammalian gastrointestinal tract is home to a complex and dynamic community
of microorganisms, termed the gut microbiota, which is essential for maintaining
host health {BÃ¤ckhed, 2005 #834}. There are complex interactions among bacterial
populations in the gut that have an important effect on host health {Levy, 2013
#3476;Marino, 2014 #3290;Lepp, 2004 #844}. The number of diseases that are
associated with abnormalities in the gut microbiota highlights the importance of
these ecological interactions {Turnbaugh, 2006 #1519;Tamboli, 2004
#3477;Saulnier, 2011 #3478}. Over the last several years, it has been well
documented that perturbations to this community are associated with colorectal
cancer (CRC) in humans and mice {Chen, 2013 #2734;Chen, 2012 #2684;Kostic, 2011
#2435;Geng, 2013 #2686;Shen, 2010 #2687;Sobhani, 2011 #2444;Wang, 2012
#3286;Ahn, 2013 #3263}. We have previously shown that CRC-associated changes in
the gut microbiota directly potentiate colon tumorigenesis in a mouse model of
CRC {Zackular, 2013 #3305}. In that study we observed clear shifts in the
microbiota that were associated with a stepwise progression in the number of
tumors that developed in the colon. In addition, we showed that transfer of the
tumor-associated microbiota to germ-free mice resulted in increased tumor
formation relative to germ-free mice that received the microbiota of healthy
mice. These results were supported by a subsequent study where we colonized
germ-free mice with the microbiota of human donors and observed that different
starting communities yielded significant variation in the number of tumors that
formed {Baxter, 2014 #3469}. Combined, these results demonstrate that the
microbiota interact with the host to affect tumor susceptibility.  A critical
question that remains unanswered is what factors and ecological principles
mediate the gut microbiota's influence on tumor development. Deciphering how
changes in microbial community composition and structure alters gut homeostasis,
and subsequently modulate tumorigenesis, is an essential step in understanding
the etiology of CRC.

Several bacterial populations including *E. coli*, *Bacteroides fragilis*, and
*Fusobacterium nucleatum* have been shown to directly influence tumor development
in the colon. The mechanisms by which bacteria potentiate these processes range
from the production of carcinogenic toxins {Arthur, 2012 #2681;Sears, 2008
#3265} to direct manipulation of the inflammatory status of the tumor
microenvironment {Kostic, 2013 #3285;Rubinstein, 2013 #3264}. Although
individual bacterial populations undoubtedly modulate colorectal
carcinogenesis, there are likely a myriad of commensal bacteria that work
together to influence tumorigenesis in the colon. This is supported by several
studies that have explored the gut microbiota associated with individuals with
CRC {Chen, 2013 #2734;Chen, 2012 #2684;Kostic, 2011 #2435;Geng, 2013
#2686;Shen, 2010 #2687;Sobhani, 2011 #2444;Wang, 2012 #3286;Ahn, 2013
#3263;Zackular, 2014 #3457}. With each study, the number of CRC-associated
bacterial populations that likely play a role in tumorigenesis continues to
grow. This is likely due to the fact that there is significant functional
redundancy within the gut microbiota and various bacterial populations may fill
similar roles in tumorigenesis {Lepage, 2013 #3479;Turnbaugh, 2009 #2387;Qin,
2010 #2337}. Furthermore, some bacterial populations have been hypothesized to
be protective against CRC {Louis, 2009 #3480;Appleyard, 2011 #3494}. This
protective phenotype may be mediated through metabolite production, induction
of immunotolerance, or an ability to outcompete pathogenic bacteria {Zhu, 2011
#3497}. We hypothesize that multiple bacteria in the gut microbiota have the
potential to play pro-tumorigenic or tumor-suppressive roles; thus, the gut
microbiota's influence on CRC is likely to be driven by complex interactions
within the microbiota and the colonic epithelium.

We have shown that conventionally-raised mice treated with a cocktail of
metronidazole, streptomycin, and vancomycin in their drinking water had a
significant decrease in tumor numbers using an inflammation-based model of CRC
{Zackular, 2014 #3457}. In the current study, we explored how differential
alterations in the microbiota by different antibiotic treatments affected the
composition of the microbiota and how changes in bacterial community structure
affected tumor susceptibility. Our results confirmed our hypothesis that the
microbiota is capable of driving tumorigenesis and that an antibiotic-based
intervention during tumor induction can arrest tumorigenesis. Our analysis
further supports the model that individual bacterial populations play an
important role in CRC, but the ecological interactions and community structure
of the gut microbiota mediate the capacity to modulate tumorigenesis.



**Results**

```{r amova}
amova_report_name <- "data/process/ab_aomdss.trim.contigs.good.unique.good.filter.unique.precluster.pick.pick.pick.an.unique_list.thetayc.0.03.lt.ave.day0.amova"

amova_report <- scan(amova_report_name, what="", sep="\n", quiet=TRUE)

headers <- amova_report[grep("\tWithin\tTotal", amova_report)]
comparisons <- gsub("^(.*)\tAmong.*", "\\1", headers)

p_value_lines <- amova_report[grep("p-value: ", amova_report)]
p_values <- as.numeric(gsub("p-value: <?([^*]*)\\*?", "\\1", p_value_lines))

names(p_values) <- comparisons
p_values <- p_values[-1]
adj_p_values <- p.adjust(p_values, method="BH")

result <- names(p_values)[adj_p_values > 0.05]
```

`r if(length(result)==0){ print("ERROR")}`

***Antibiotic perturbation of the gut microbiota modulates tumorigenicity.***
We subjected specific pathogen-free (SPF) C57BL/6 mice to an inflammation-based model of colorectal cancer that utilizes azoxymethane (AOM) as a mutagen and dextran sodium sulfate (DSS) to induce inflammation {Zackular, 2013 #3305;Baxter, 2014 #3469;De Robertis, 2011 #3300} (Figure 1A). To determine how differential changes in the gut microbiota affected tumorigenesis, we manipulated the microbiota by administering seven different antibiotic combinations for the length of the model (Interveneion 1; Figure 1A) and then quantified the effects of the treatments on the number of tumors observed at the end of the model (Figure 1BC). Specifically, we treated mice with (i) no antibiotics, (ii) metronidazole, streptomycin, and vancomycin (all antibiotics), (iii) streptomycin and vancomycin ($\Delta$ metronidazole), (iv) metronidazole and vancomycin ($\Delta$ streptomycin), (v) metronidazole and streptomycin ($\Delta$ vancomycin), (vi) metronidazole, (vii) streptomycin, and (viii) vancomycin. The three antibiotics were selected based on their reported ability to target general groups of bacteria including anaerobes (metronidazole), Gram-negatives (streptomycin), and Gram-positives (vancomycin). Upon necropsy we observed that perturbation of the microbiota through the use of antibiotics yielded a differential capacity for colon tumorigenesis (Figures 1BC). Sequencing the 16S rRNA genes that were present in the feces of conventional and antibiotic-treated mice demonstrated that the different antibiotic treatments generated different bacterial communities prior to AOM injection (Figure 1DE); however, the composition of these communities could not have been predicted by the spectrum of the antibiotic that was used to treat the mice. The eight community structures generated by using the untreated mice and those that received one of the seven antibiotic combinations were all significantly different from each other (all P<0.05 by AMOVA with Benjimani-Hochberg correction). These results and an ordination of the communities indicated that the communities are distinct from each other (Figure 1E) and varied in their ability to drive tumorigenesis.



```{r predict_tumor_baseline}
load("data/process/baseline_model.Rdata")

number <- c("one", "two", "three", "four", "five", "six", "seven", "eight", "nine", "ten", "eleven", "twelve")
```


***Tumor burden can be predicted from the initial microbiota.***
Tumor burden can be predicted from the initial microbiota. Serial collection of fecal samples allowed us to ascertain the composition of the microbiota for each mouse and associate it with the number of tumors that developed at the end of
the model. Using the 16S rRNA gene sequence data generated from feces collected on the day of AOM injection, we assigned the sequences to operational taxonomic units (OTUs) that were defined as a group of sequences that, on average, were not more than 3% different from each other. We then used the regression-based random forest machine learning algorithm to identify OTUs that would enable us to predict the number of tumors that developed at the end of the model. The model that included all `r n_full_features` OTUs explained `r round(100*baseline_all_rsq, 1)`% of the variation in the tumor counts. We then sorted the OTUs by their importance in the random forest model as determined by the percent reduction in the mean squared error when that OTU was removed from the model and observed a peak in the amount of the variation explained in the observed tumor counts when we used the `r number[nrow(baseline_features)]` most important OTUs (Supplemental Figure 1). The simplified model explained `r round(100*baseline_top_rsq, 1)`% of the variation in the observed tumor counts (Figure 2). These `r number[nrow(baseline_features)]` OTUs included members of the Firmicutes (OTUs 1, 66, 99, and 185), Bacteroidetes (OTUs 14, 67, 79, and 107), Proteobacteria (OTUs 7, 36, and 72), and Tenericutes (OTU 9). With the exception of OTUs affiliated with members of the *Lactobacillus* (OTU 1) and Betaproteobacteria (OTU 7), each of the OTUs was associated with increased tumor burden (Figure 3). The relative abundance of the *Lactobacillus*-affiliated OTU at the start of the model was inversely proportional to the tumor burden at the end of the model. There was not a clear relationship between the initial relative abundance of the *Betaproteobacteria*-affiliated OTU and the final tumor burden. Interestingly, tumorigenesis was not dependent on the presence of any of the OTUs. In other words, tumors could form even when an OTU was below the limit of detection. This suggests that the role of the microbiota in driving tumor formation was context dependent. More broadly, our random forest modeling demonstrated that it was possible to predict the number of tumors at the end of the model based on the composition of the microbiota at the beginning of the model.

```{r predict_tumor_final}
load("data/process/final_model.Rdata")
```

***Tumor burden can be predicted from the microbiota at the end of the model.***
Similar to our analysis using the initial composition of the microbiota, we developed a random forest regression model to predict the number of tumors in the mice based on the composition of the microbiota at the end of the model (Supplemental Figure 2). The simplified model included `r number[nrow(final_features)]` OTUs. The simplified model explained `r round(100*final_top_rsq, 1)`% of the variation in the tumor counts (Supplementary Figure 3), which was comparable to what we observed when we modeled tumor counts based on the initial community composition. This model utilized the relative abundance data from OTUs affiliated with members of the Firmicutes (OTU 85), Bacteroidetes (OTUs 7, 19, 28, 29, and 51), Proteobacteria (OTU 7), and candidate phylum Saccharibacteria (OTU 192; Supplementary Figure 4). Interestingly, of the OTUs that were predictive of the number of tumor counts using the initial and final community composition data, only `r number[length(intersect(rownames(final_features), rownames(baseline_features)))]` of the OTUs overlapped, which was affiliated with the Betaproteobacteria (OTU 7). The distinction between OTUs that were predictive of tumor burden using the community composition at the beginning and end of the model suggests that the communities that gave rise to tumors were different from those that were enriched in a tumor-laden environment.

```{r distance_tumor_correlation}
load("data/process/distance_tumor_correlation.Rdata")
```

***The microbial community is dynamic during inflammation-associated tumorigenesis.***
Using mice that were colonized with human feces, we previously reported that tumor burden was associated with the amount of change in the community structure over the course of the AOM-DSS model {Baxter, 2014 #3469}. In the current study, however, there was a non-significant association between the change in the community structure as measured by the $\theta$<sub>YC</sub> metric of community structure similarity and tumor burden ($\rho$=`r round(dist_tumor_cor$estimate,2)`, P=`r round(dist_tumor_cor$p.value, 2)`; Figure 4A). We did observe that mice that did not receive antibiotics and those that received the Îvancomycin and Îmetronidazole treatments changed the most over the course of the model. When we identified those OTUs whose relative abundances changed the most across each treatment group, we found that OTUs affiliated with the *Lactobacillus* (OTU 1) and Enterobacteriaceae (OTU 2) were consistently among the most dynamic OTUs across the treatment groups (Figure 4). Interestingly the initial relative abundance of the *Lactobacillus*-affiliated OTU was predictive of the final tumor burden, but the final relative abundance of neither OTU was predictive of final tumor burden. These data suggest that the magnitude of change that occurs across a microbial community during tumorigenesis is not strongly associated with tumor burden. Instead, the relative abundance of a subset of populations within the community dictate tumor burden.


***Antibiotic intervention during inflammation reduces tumorigenesis.***
The results of our current study and our previous investigations of the role of the gut microbiota in colonic tumorigenesis have suggested that by manipulating the gut microbiota, it would be possible to manipulate tumorigenesis {Zackular, 2013 #3305;Baxter, 2014 #3469}. To fourther validate these results, we performed two additional antibiotic intervention experiments. We first treated mice with the vancomycin, metronidazole and streptomycin two weeks prior to the administration of AOM and up until the first round of DSS and then removed the antibiotic cocktail for the remainder of the model (Intervention 2; Figure 1A). We found that these mice had a similar tumor burden to untreated mice (Figure 5).  Next, we treated mice after the first round of DSS administration with the antibiotic cocktail until the end of the model. Our previous work found that the period following the first round of DSS coincided with a period when inflammatory responses were the greatest and there were aberrant changes in the gut microbiota {Zackular, 2013 #3305} (Intervention 3; Figure 1A). With these mice, we found that the intervention resulted in a significant decrease in the number of tumors (Figure 5). These results suggest that the gut microbiota-mediated effect on CRC is independent of AOM-mediated carcinogenesis. Furthermore, it shows that targeting the gut microbiota at later stages of tumor growth is a viable option for minimizing tumorigenesis and highlights microbiota manipulation as a potential therapeutic in CRC.


**Discussion**

In the present study, we established the importance of the microbial community
structure in determining the extent of tumorigenesis.  We demonstrated that
manipulation of the murine gut microbiota with different antibiotic cocktails
resulted in distinct community structures that were associated with
disparate levels of tumorigenesis. To determine whether the microbiota was involved in possibly converting the AOM to a carcinogenic metabolite or involved in the inflammation process we restricted the application of antibiotics to alter the microbiota during these phases of the model. We determined that the gut microbiota affects tumorigenesis via a mechanism that does not involve AOM-induced carcinogenesis. Our experiments also demonstrated that targeting the gut microbiota at the emergence of dysbiosis (i.e. after the first round of DSS in the AOM/DSS model) is a viable strategy for the amelioration of colon tumorigenesis. Such a result offers hope that by altering a person's gut microbiota it may be possible to alter their risk of developing colon cancer.

Our analysis suggests that community-wide changes affect the process of tumorigenesis in the murine gut. To investigate this process, we manipulated the gut microbiota by applying various antibiotic cocktails. One risk of this approach is that the antibiotic perturbation could reduce the overall bacterial load. We previously analyzed the feces of mice receiving all three antibiotics using a culture-independent quantitative PCR approach and observed a non-significant reduction in the bacterial load {Zackular, 2014 #3457}. This result is in agreement with other studies that have used a similar antibiotic cocktail in studying *Salmonella* infection and colitis {Sekirov, 2008 #3507;Garrett, 2007 #3508}. Meanwhile, others have seen a small but significant decrease in bacterial load which varied along the gastrointestinal tract {Ubeda, 2010 #3509}. Considering our previous result and the fact that we observed we observed a relatively consistent relationship between bacterial populations in the gut and tumor burden, it is unlikely that differences in the bacterial load of the colon is responsible for the observed results. An alternative approach would involve colonizing germ-free mice with defined cocktails of bacteria or from murine or human donors. The challenge of this approach is that the immune system would still be altered from a normal state and it is difficult to dictate the final structure of a transplanted community {Baxter, 2014 #3469}. By pursuing various approaches to generate variation in the initial community, it is clear that gut microbiota are involved in protecting against and exacerbating colonic tumorigenesis.

In recent years, there has been a focus on identifying specific bacterial populations that are etiologic agents of CRC. Several commensal bacteria, including *E. coli*, *Fusobacterium nucleatum* and enterotoxigenic *Bacteroides fragilis* (ETBF) have been linked to CRC in humans {Arthur, 2012 #2681;Rubinstein, 2013 #3264;Sears, 2008 #3265}. *F. nucleatum*, which has been detected on the surface of over 50% adenomas in one study, can promote inflammation within the tumor microenvironment in multiple intestinal neoplasia mice {Kostic, 2013 #3285;Kostic, 2011 #2435}. ETBF increases tumor multiplicity in the colon of multiple intestinal neoplasia mice through the action of a secreted metalloprotease toxin. It has been estimated that between 5-35% of people carry ETBF {Housseau, 2010 #3498}. Although there is substantial evidence for a role in potentiating tumorigenesis, the fact that each of these bacteria is only associated with a fraction of CRCs suggests that it is unlikely that there is a single microbial agent that causes cancer. Rather, the role of the gut microbiota in CRC is likely polymicrobial in nature. The results in the present study support this hypothesis, as we demonstrated that different community structures were associated similar levels of tumorigenesis in mice. When we examined the relative abundance of bacterial populations associated with increased tumor burden, we never observed consistent enrichment of any one population across all treatment groups (Figure 3). Similar to a previous study exploring the role of the gut microbiota in shaping resistance against *C. difficile* colonization {Schubert, 2015 #3510}, we found that the context of the gut microbiota is important in predicting the eventual tumor burden. Such a result suggests that there may be redundancy in tumor-modulating roles amongst different bacteria populations within the gut microbiota.

As described above, there has been considerable effort to identify bacteria and their products that cause colon cancer. In contrast, our results indicate a need to focus on protective populations. We consistently observed that that the relative abundance of a *Lactobacillus*-affiliated OTU (OTU 1) was predictive of a low tumor burden (Figure 3). *Lactobacillus* are widely used as probiotics to reduce inflammation in the gastrointestinal tract. These bacteria have been successfully reduced colonic inflammation in mouse models of colitis {Chen, 2013 #3512}, necrotizing enterocolitis {Liu, 2014 #3514}, and graft-versus-host disease {Jenq, 2012 #3513}. It has been shown that *Lactobacillus* enhances epithelial barrier function by inducing the production of mucus and tight junction proteins {Mack, 2003 #3515;Liu, 2011 #3516} and that it can modulate the host's immune response by suppressing the expression of the proinflammatory cytokine IL-17 {Chen, 2015 #3511}. The clinical significance of this result is unclear, however, considering we observed suppression of tumorigenesis when the microbiota had levels of *Lactobacillus* that were higher than the 0.1 to 1% relative abundance commonly observed in the feces of humans {The Human Microbiome Consortium, 2012 #2617}. Regardless, a better understanding of the possible protective role of *Latobacillus* in limiting tumorigenesis may be useful in developing probiotic and prebiotic therapies.

It is striking that we were able to quantitatively predict the tumor burden that resulted at the end of our 73-day model based on the community composition at the start of the model. The random forest regression modeling approach is non-parametric and accounts for the nonlinearities and interactions within the dataset to identify a subset of OTUs that are predictive of tumor burden. An added advantage of this approach is that cross-validation is built into the model generation procedure limiting the risks of overfitting the model to the data {Breiman, 2001 #2526}. Although random forest models are commonly used for classifying samples to specific categories, the regression-based approach has been used with microbiome data to predict *C. difficile* colonization {Schubert, 2015 #3510} and to assign a microbiome-based age to malnourished children {Subramanian, 2014 #3517}. Given the significant heterogeneity that we observe in gut microbiota, regression-based random forest models are a powerful tool to identify subsets of communities that are associated with disease.

In our previous work, we demonstrated that dysbiosis of the gut microbiota
generates a pro-inflammatory environment which results in a self-reinforcing
pathogenic cascade between the gut microbiota and the host {Zackular, 2013
#3305;Baxter, 2014 #3469}. In this study, we demonstrated that antibiotic
manipulation of the gut microbiota during the onset of inflammation can
significantly decrease tumorigenesis in mice. This highlights the efficacy of
targeting the gut microbiota in CRC. Additional studies are needed to explore
the viability of manipulating the gut microbiota in CRC with methods such as
diet, probiotics, and prebiotics.



**Materials & Methods**

**Animals and animal care.** Studies were conducted using adult (8 to 12 week
old) age-matched C57BL/6 male mice that were maintained under SPF conditions.
Mice were co-housed in groups of five and fed the same autoclaved chow diet. All
animal experiments were approved by the University Committee on Use and Care of
Animals at the University of Michigan and carried out in accordance with the
approved guidelines.

**Inflammation-induced colon tumorigenesis.** Mice received a single
intraperitoneal (i.p.) injection of azoxymethane (10 mg/kg). Water containing
2% DSS was administered to mice beginning on day 5 for 5 days followed by 16
days of water. This was repeated twice for a total of 3 rounds of DSS
{Zackular, 2013 #3305}. Mice were euthanized 3 weeks after the third round of
DSS administration for tumor counting. At necropsy, all colons were harvested,
flushed of luminal contents, and cut open longitudinally to count and measure
tumors.

**Antibiotic treatment.** Mice were treated with all possible combinations of
metronidazole (0.75 g/L), streptomycin (2 g/L), and vancomycin (0.5 g/L) to
create eight treatment groups: no antibiotics (N=12), all antibiotics (n=9)
(metronidazole, streptomycin, and vancomycin), $\Delta$ metronidazole (n=5)
(streptomycin and vancomycin), $\Delta$ streptomycin (n=5) (metronidazole and
vancomycin), $\Delta$ vancomycin (n=5) (metronidazole and streptomycin), metronidazole
only (N=5), streptomycin only (N=5), and vancomycin only (N=3). Antibiotics were
administered in mouse drinking water for 2 weeks prior to and throughout the
duration of AOM/DSS administration, unless otherwise specified in Figure 1A.
Tumors were enumerated at the end of the model.

```{r error_rate}
error <- read.table(file="data/process/ab_aomdss.trim.contigs.good.unique.good.filter.unique.precluster.pick.pick.pick.error.summary", header=T, row.names=1)
error_nochim <- error[error$numparents==1,]
error_rate <- 100 * sum(error_nochim$weight*error_nochim$mismatches) /
                    sum(error_nochim$weight*error_nochim$total)
```

**DNA extraction and 16S rRNA gene sequencing.** Fecal samples were collected
daily from the mice throughout the AOM/DSS protocol and immediately frozen for
storage at -20Â°C. For each mouse, 8 fecal samples distributed over the 73-day
timeline of the AOM/DSS model were selected for analysis (Figure 1A). Microbial
genomic DNA was extracted using the PowerSoil-htp 96 Well Soil DNA Isolation Kit
(MO BIO laboratories) using an EpMotion 5075. The V4 region of the 16S rRNA gene
from each sample was amplified, sequenced using the Illumina MiSeq Personal
Sequencing platform, and curated as described previously using the mothur
software package {Kozich, 2013 #2719; Schloss, 2009 #1816}. Briefly, we reduced
sequencing and PCR errors by requiring reads to fully overlap and in cases where
base calls conflicted, we broke the conflict by requiring one base call to have
a PHRED quality score 6 units higher than the other otherwise the base call was
replaced with an ambiguous base call in the contig. Any reads containing
ambiguous base calls were culled. Sequences were aligned to a customized version
of the SILVA 16S rRNA sequence database {Pruesse, 2007 #1735} and were screened
to insure that they correctly overlapped within the V4 region. Chimeric
sequences were identified using the de novo implementation of UCHIME and they
were culled {Edgar, 2011 #2406}. The resulting sequences had a median length of
253 nt and we rarefied to 2,500 sequences per sample to limit effects of uneven
sampling. A mock community was sequenced and processed in parallel to the fecal
samples. Based on the mock community data we observed a sequencing error rate of
`r round(error_rate, 2)`%. The complete analysis methods and this document as an R-executable
document are available at
https://github.com/SchlossLab/Zackular_AbAOMDSS_SciReports_2015. All FASTQ
sequence data can be obtained from the Sequence Read Archive at NCBI (Accession SRP056144).

**Statistical analysis.** The microbiota data were analyzed using the R project
for statistical computing. All R source code is available on our GitHub
repository at https://github.com/SchlossLab/Zackular_AbAOMDSS_SciReports_2015.
All random forest models were made using the randomForest package with 10,000
trees {Breiman, 2001 #2526}. Diagnostic plots indicated that the percent of the
variance explained had stabilized with this number of trees. Comparison of
tumor counts were made by carrying out non-parametric pairwise Wilcoxon tests.
The resulting p-values were corrected for multiple comparisons using the
Benjamini-Hochberg procedure using an experiment-wide Type I error rate of 0.05.


**Acknowledgements**

This work was supported by grants from the National Institutes for Health to PDS
(R01GM099514, R01HG005975, P30DK034933, University of Michigan GI SPORE) and GYC
(University of Michigan GI SPORE, ARRA Supplement P30CA4659-22S3, and R01
CA166879).


**References**








**Contributions**
All authors contributed to the design of the experiments. JPZ and NTB carried
out the experiments and generated the data. JPZ and PDS analyzed the data. All
authors participated in interpreting the results. JPZ and PDS wrote the
manuscript and NTB and GYC helped with the final editing of the text.


**Competing financial interests**
The authors declare no competing financial interests.



**Figure legends**

```{r stress}
stress_table <- read.table(file="data/process/ab_aomdss.trim.contigs.good.unique.good.filter.unique.precluster.pick.pick.pick.an.unique_list.thetayc.0.03.lt.ave.day0.nmds.stress", header=T)
best <- which.min(stress_table$Stress)
stress <- stress_table[best, "Stress"]
fit <- stress_table[best, "Rsq"]
```
**Figure 1. Antibiotic perturbation drives changes in microbial community structure and final tumor burden.** The AOM-DSS model was administered to C57BL/6 mice reared under standard pathogen free (SPF) conditions with different antibiotic perturbations which were applied during the period covered by each of the rectangles; Black arrows indicate fecal samples that used for our analysis (A). The mice were treated with all possible combinations of metronidazole, streptomycin, and vancomycin to create eight treatment groups, which resulted in a continuum of tumor burden in the mice (C and D). The stars indicate which treatments yielded a significantly (P<0.05) different number of tumors when compared to the treatment with the vertical line. The antibiotic treatments resulted in variation in the taxonomic structure of the communities at the start of the model (Day 0) (D). The two dimensional NMDS ordination had a stress of `r format(stress, digits=2, nsmall=2)` and explained `r format(fit*100, digits=1, nsmall=1)`% of the variation in the distances (E).


**Figure 2. A random forest model successfully predicted the number of tumors in the mice at the end of the model based on their microbiota composition at the start of the model.** The model explains XX.X% of the variation in the data.


**Figure 3. Relationship between the initial relative abundance of the most informative OTUs from the random forest model with the number of tumors found in the mice at the end of the model.** The vertical gray line indicates the limit of detection. Panels are ordered in decreasing order of the percent increase in the mean squared error of the model when that OTU was removed. The color and shape of the plotting symbols corresponds to those used in Figure 2.


**Figure 4. The murine microbiota is dynamic but the amount of change is
not associated with the final number of tumors.** The structure of the gut
microbiota associated with untreated and the $\Delta$ metronidazole and $\Delta$
vancoymcin-treated mice changed the most throughout the model as measured using
the $\Theta$<sub>YC</sub> distance metric (A). OTUs 1 and 2 were among the
most dynamic OTUs across all treatment groups; here we depict the change in their relative abundance across the model for those treatment groups that experienced the greatest overall change in community structure (B). The plotting symbols and characters are the same as those used in
Figure 1. In panel B, the median relative abundance is indicated by the plotting
symbol and the range of observed relative abundances is plotted by the vertical
bar. The vertical blue regions indicate when the DSS treatments were applied.


**Figure 5. Antibiotic intervention prior to second administration of
DSS alleviates tumor burden.** Interventions with an antibiotic cocktail
of metronidazole, vancomycin, and streptomycin were performed as
depicted in Figure 1A with enumeration of tumors performed at the end point of
the model (A). Representative images of tumors in the distal colon of mice from
each treatment group (B).


**Supplemental Figure 1. Effect of pruning the number of OTUs included in the random forest model for predicting the number of tumors at the end of the model based on the microbiota found at the start of the model.** The order of OTUs was set by the percent increase in mean square error when that OTU was removed from the model. The percent of the variance explained here indicates the quality of the fit when the top features were used to generate a model. The star indicates the number of OTUs that resulted in the model explaining the maximum percent of the variance.


**Supplemental Figure 2. Effect of pruning the number of OTUs included in the random forest model for predicting the number of tumors at the end of the model based on the microbiota found at the end of the model.** The order of OTUs was set by the percent increase in mean square error when that OTU was removed from the model. The percent of the variance explained here indicates the quality of the fit when the top features were used to generate a model. The star indicates the number of OTUs that resulted in the model explaining the maximum percent of the variance.


**Supplemental Figure 3. A random forest model successfully predicted the number of tumors in the mice at the end of the model based on their microbiota composition at the end of the model.** The model explains XX.X% of the variation in the data.


**Supplemental Figure 4. Relationship between the final relative abundance of the most informative OTUs from the random forest model with the number of tumors found in the mice at the end of the model.** The vertical gray line indicates the limit of detection. Panels are ordered in decreasing order of the percent increase in the mean squared error of the model when that OTU was removed. The color and shape of the plotting symbols corresponds to those used in Figure S3.
